### PR TITLE
Option to disable controls and enable autoplay on video

### LINKED ISSUE #1485: Option to disable controls on video and audio bubbles

### PR DESCRIPTION
**Pull request by basuabhirup**
Extends the implementation of #1503 as per the suggestions provided in the code review to resolve #1485 

https://github.com/baptisteArno/typebot.io/assets/69730155/87481d64-57f5-4f7e-8a28-4a464f12cc31

### CODE REVIEW

File: packages/schemas/features/blocks/bubbles/video/constants.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ export const defaultVideoBubbleContent = {
   height: 400,
   aspectRatio: '16/9',
   maxWidth: '100%',
+  areControlsDisplayed: true,
+  isAutoplayEnabled: false,
```
Comment (baptisteArno) [inline packages/schemas/features/blocks/bubbles/video/constants.ts:21]: Should be `true` by default

**Review by baptisteArno — CHANGES_REQUESTED** [review]
Thanks 👍

File: packages/schemas/features/blocks/bubbles/video/constants.ts (hunk 1 above)
Comment (basuabhirup) [reply packages/schemas/features/blocks/bubbles/video/constants.ts]: Updated


https://github.com/baptisteArno/typebot.io/assets/69730155/917386b8-66ae-45fc-b12d-06c4cfeb86ed

**Review by baptisteArno — APPROVED** [review]

**Comment by baptisteArno** [comment]
Good job mate, that's great 👏

**Pull request merged**